src/paging.rs

```diff
@@ -25,6 +25,17 @@ pub const PAGE_SIZE: usize = 1 << PAGE_SHIFT;
 /// page size.
 pub const BITS_PER_LEVEL: usize = PAGE_SHIFT - 3;
 
+/// Which TTBR register to use for a page table.
```

Comment:
Maybe it's easiest to use the register names, but you could alternatively be slightly more abstract and  refer to the lower or upper VA range; that's the ARM ARM terminology.

Reply:
Done.